### ISSUE TITLE
Is this working?

### ISSUE THREAD

**Issue by mihle** [cloudops/go-xen-console-proxy#1]
Hello, I an very interested in getting a noVNC replacement for the console vm for CloudStack.  This looks great - but before I go down the rabbit hole - is it working?  I see no other pull requests and installation documentation is sparse.  Please let me know.  Thanks!

**Comment by swill** [comment]
@syed can you comment on this?  I know we have not deployed it into production yet, but is the majority of the functionality in place?  I don't remember if there was anything outstanding with this implementation.

**Comment by syed** [comment]
Yes this works. However, I've only tested this on XenServer. I was running this on my dev setup. There are still things that need to be ironed out like how it would play with the existing console which provides RDP as well. There was also a bit of hackery involved on Cloudstack to enable this. But there was a recent patch by someone at ENA which would make that moot. 

Let me know if you are interested in taking this further. I will be happy to help you getting started.

**Comment by mihle** [comment]
Thanks for the reply!  I will go ahead and try it in my dev environment and let you know my results.

**Comment by syed** [comment]
Remember you need changes on the Cloudstack side as well. Once you have a setup, let me know and I can give you that patch
